CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
+   build.
```

Comment:
Seems like this is talking about Docker? It's not totally clear...

Reply:
Since we don't have Docker involved yet, I'll just clarify this as ensuring all dependency changes are captured.